README.rst

```diff
@@ -23,12 +23,12 @@ repository:
 `readthedocs/build:3.0`
     ``stable``
     Ubuntu 16.04 supporting Python 2.7, 3.3, 3.4, 3.5, and 3.6.
-    This is the **stable** default image supported by Read the Docs.
+    This is the **stable** image supported by Read the Docs.
 
 `readthedocs/build:4.0`
     ``latest``
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7.
-    This is the **latest** default image supported by Read the Docs.
+    This is the **latest** default image used for documentation builds and supported by Read the Docs.
```

Comment:
All images are used for builds in read the docs

Reply:
We need to have documented way to emulate readthedocs build process, to test documentation builds locally. Some builds work on our infrastructure, but don't work on the readthedocs. Thus why I've asked for this change.

Reply:
The images are only supported to build inside rtd, you can check https://github.com/rtfd/readthedocs.org/issues/5325#issuecomment-465783090 to try locally.